guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -13,18 +13,28 @@ You only need the MAC address for the BMC interface if the BMC interface is mana
 ====
 
 .Procedure
-. Enable BMC on the {SmartProxy} server if it is not already enabled:
-.. Configure BMC power management on {SmartProxyServer} by running the `{foreman-installer}` script with the following options:
+. Install `ipmitool` on your {SmartProxy}:
```

Comment:
I am not sure why you moved the ipmitool installation to the procedure. I would have kept it in the prerequisites. The installation process is fairly simple, so I think the user could figure out how to install the tool without the command.

Reply:
In other places, like the Provisioning Hosts guide, we include package installations in the procedures. I don't mind moving it.

Reply:
I am OK either way. I lean towards "procedure" because it feel likes "just another step" to do.